package.json

```diff
@@ -5,7 +5,7 @@
   },
   "name": "mSupplyMobile",
   "//": "version must be in the format ${majorNumber}.${minorNumber}.${patchNumber}-rc${releaseCandidateNumber}",
-  "version": "8.2.1",
+  "version": "8.2.2",
```

Comment:
Wait, shouldn't this also be `8.2.3`?

Reply:
Plan was for the version to be bumped in `master` after merging.

Reply:
Cool beans!